core/lib/libtomcrypt/src/ciphers/sub.mk

```diff
@@ -1,5 +1,13 @@
 cflags-y += -Wno-unused-parameter
 
+ifeq ($(CFG_CRYPTO_AES_ARM64_CE),y)
+srcs-y += aes_arm64_ce.c
+cflags-aes_arm64_ce.c-y += -march=armv8-a+crypto
```

Comment:
That is a very valid point, indeed `+crypto` implies `+simd` which implies FP instructions are enabled. I guess I'd better put the functions that have inline CE instructions in a separate compilation unit.